tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -348,3 +348,42 @@ async def test_restarting_f_replicas_for_view_change(self, bft_network):
         await bft_network.wait_for_fast_path_to_be_prevalent(
             run_ops=lambda: skvbc.run_concurrent_ops(num_ops=20, write_weight=1), threshold=20)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: c == 0, rotate_keys=True)
```

Comment:
`c==0` is the default one now, so no need to specify it:
https://github.com/vmware/concord-bft/blob/master/tests/apollo/util/bft.py#L95

Reply:
done